other/run-examples.js

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+
+const { exec } = require('child_process')
+const fs = require('fs')
+const path = require('path')
+const examplesFolder = 'packages/data-point/examples/'
+
+fs.readdir(examplesFolder, (err, files) => {
+  if (err) {
```

Comment:
if there are errors the script will not warn any outsider about it, can you add process.exit(1) so its interpreted as an error, this will allow us to chain this script with others as well